planer-uwr-webapp/imports/ui/pages/PlanPage/RuleEntry.tsx

```diff
@@ -32,6 +32,15 @@ const RuleEntry = (props: RuleEntryProps) => {
                     <Separator value="" />
                 </span>
             );
+            case 'Proseminarium':
+                return (
+                    <span>
+                        <CourseTypeTag courseType={courseTypeById['14']} />
+                        <Separator value="+" />
+                        <CourseTypeTag courseType={courseTypeById['41']} />
+                        <Separator value="" />
+                    </span>
+                );
```

Comment:
Thanks for this addition. Could you just fix indentation in this place so it matches the code around?

Reply:
Fixed